code/modules/modular_armor/attachables.dm

```diff
@@ -50,7 +50,7 @@
 
 /obj/item/armor_module/attachable/better_shoulder_lamp/mark1
 	name = "\improper Mark 1 Baldur Light Amplification System"
-	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight. Slows you down minorly."
+	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight."
```

Comment:
Due to inheritance this change is incorrect as it inherits the slowdown from its parent
However you could instead change it back and note that it increases light radius by less

Reply:
Can do